### PR TITLE
create server

### PR DESCRIPTION
**Pull request by meisen45**

### CODE REVIEW

File: server/androidtv_server.py [hunk 1]
```diff
@@ -0,0 +1,753 @@
...
+    source = ndb.StringProperty(indexed=False, required=True)
+    card = ndb.StringProperty(indexed=False, required=True)
+    background = ndb.StringProperty(indexed=False, required=True)
+    title = ndb.StringProperty(indexed=False, required=True)
+    studio = ndb.StringProperty(indexed=False, required=True)
+
+
+# global variable to track database status
+# when the database is created, this flag will be toggled to true
+DATABASE_CREATED = False
```
Comment (dakegu2) [inline server/androidtv_server.py]: Does this global variable get lost when app server is restarted?

Comment (meisen45) [reply server/androidtv_server.py]: Yes, you are absolutely right. So I removed this global variable, another global variable VIDEO_ID will be updated every time through an explicit get request to url/create

File: server/androidtv_server.py [hunk 2]
```diff
@@ -0,0 +1,809 @@
...
+# handler to get all videos in same category
+# the category information will be passed from the url
+class GetVideosInSameCategory(webapp2.RequestHandler):
+    def get(self):
+        # add artificial random latency
+        time.sleep(generate_random_latency())
+
+        category = self.request.get('category')
+        # we will randomize the database every time when the request is received by server
+        if random.random() > 0.5:
```
Comment (dakegu2) [inline server/androidtv_server.py]: can we remove this random stuff?  for testing, use another request url e.g. 

shuffle_category?name=xyz
dup_category?from=xyz&to=abc

Comment (meisen45) [reply server/androidtv_server.py]: Yes, it makes sense that avoid random factor from the server side. 

Currently, the server has different urls to duplicate/ shuffle the channel, client can hit different end point randomly.

**Pull request merged**